### PR TITLE
FIX:  warnings on gcc v7.2 ubuntu

### PR DESCRIPTION
**Pull request by iceman1001**
This fixes 
- warning DBG
- warning too many arguments for format 

when compiling on 
**gcc version 7.2.0 (Ubuntu 7.2.0-1ubuntu1~16.04)** 

```
slre.c:37:0: warning: "DBG" redefined
#define DBG(x)
 
In file included from mfoc.c:52:0:
nfc-utils.h:58:0: note: this is the location of the previous definition
  define DBG(...) {}
```

```
mfoc.c:913:12: warning: too many arguments for format [-Wformat-extra-args]
```

### CODE REVIEW

File: src/mfoc.c [hunk 1]
```diff
@@ -910,7 +910,7 @@ get_rats_is_2k(mftag t, mfreader r)
     exit(EXIT_FAILURE);
   }
   if (res >= 10) {
-    printf("ATS %02X%02X%02X%02X%02X|%02X%02X%02X%02X\n", res, abtRx[0], abtRx[1], abtRx[2], abtRx[3], abtRx[4], abtRx[5], abtRx[6], abtRx[7], abtRx[8]);
+    printf("ATS %02X%02X%02X%02X%02X|%02X%02X%02X%02X%02X\n", res, abtRx[0], abtRx[1], abtRx[2], abtRx[3], abtRx[4], abtRx[5], abtRx[6], abtRx[7], abtRx[8]);
```
Comment (smortex) [inline src/mfoc.c]: This change is unrelated

Comment (iceman1001) [reply src/mfoc.c]: ok,  changed title.

Comment (smortex) [reply src/mfoc.c]: D'oh, I though it was a mistake… What warning is this change fixing?

Comment (smortex) [reply src/mfoc.c]: Never mind… I have to read what you have already written…

**Comment by smortex** [comment]
Looks good to me :+1:

File: src/mfoc.c (hunk 1 above)
Comment (iceman1001) [reply src/mfoc.c]: now,  I was thinking that you would comment that the added formatter was in the wrong place...
Since I basically just added it to the end,

Comment (smortex) [reply src/mfoc.c]: I have no idea what this code is supposed to do :smiley:  Nevertheless, 10 `%x` in a `printf()` require 10 arguments.
